Figure_07_02.py

```diff
@@ -32,10 +31,11 @@ def randomparameters():
     frn = G(s)
     r = abs(w_A(s))
 
-    for i in range(N):
-        k, tau, theta = randomparameters()
-        frp = G_P(k, tau, theta, s)
-        plt.plot(np.real(frp), np.imag(frp), 'b.')
+    frp = np.array([G_P(k,tau,theta,s)
```

Comment:
I don't understand the improvement here. What's wrong with randomly selected parameters?